config/locales/debt_management/debt_management.cy.yml

```diff
@@ -227,3 +230,42 @@ cy:
           <a href="tel:+443001239123">0300 123 9 123</a> o ffôn symudol</p>
           <p>Mae llinellau ar agor dydd Llun i ddydd Gwener – 8am i 8pm, dydd Sadwrn – 9am i 1pm]<br/>
           Botwm [Yn ôl i’r dudalen flaenorol] sy’n mynd â’r defnyddiwr yn ôl i’r dudalen lanio</p>
+  companies:
+    title: Pa gwmnïau yn cael eu heffeithio?
+    intro_html: |
+      <p>Mae’r cwmnïau ar y rhestr hon wedi dewis peidio ag ymgeisio am awdurdod gan yr FCA ac ni chânt gynnig cynlluniau rheoli dyledion mwyach. Byddwn yn diweddaru’r rhestr hon yn rheolaidd os bydd rhagor o gwmnïau’n penderfynu peidio ag ymgeisio am awdurdod neu ddim yn llwyddo i’w gael.</p>
+      <p>Gallai rhai cwmnïau a oedd yn darparu Cynlluniau Rheoli Dyledion fod wedi gwerthu’u rhestr gleient i gwmni arall. I sicrhau bod y cwmni yr ydych chi’n cydweithio ag ef yn dal wedi ei awdurdodi gan yr FCA, gwiriwch y Gofrestr Credyd Defnyddwyr <a href="http://fca-consumer-credit-interim.force.com/CS_RegisterSearchPageNew"> http://fca-consumer-credit-interim.force.com/CS_RegisterSearchPageNew</a>.</p>
+    standalone_title: Cwmnïau nad ydynt yn gweinyddu Cynlluniau Rheoli Dyledion mwyach
+    standalone_intro: Mae’r FCA yn gwirio pob cwmni i sicrhau eu bod yn dilyn ei reolau ac yn trin eu cwsmeriaid yn deg. Mae’n gwneud hyn drwy ofyn iddynt wneud cais i gael eu hawdurdodi. Mae rhai cwmnïau ar y rhestr hon un ai wedi dewis peidio â gwneud cais neu heb gael eu hawdurdodi.
+    back_to_faq_html: Am ragor o wybodaeth darllenwch ein <a href="/cy/campaigns/debt-management/faq">Cwestiynau Cyffredin</a>
+    companies:
+      - text: Moneyfocus Financial Solutions Limited
```

Comment:
FYI I think you can make this as a list and not have the `text` key bit, http://yaml.org/spec/1.0/#id2489726